server/bleep/migrations/20230919100529_code_studio_docs.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE docs (
+    id INTEGER PRIMARY KEY,
+
+    url TEXT NOT NULL,
+    index_status TEXT NOT NULL,
+
+    name TEXT,
+    favicon TEXT,
+    description TEXT,
+
+    modified_at DATETIME NOT NULL DEFAULT (datetime('now'))
+);
+ALTER TABLE studio_snapshots ADD COLUMN doc_context TEXT;
```

Comment:
Can we make this `NOT NULL` and migrate existing snapshots to have a `doc_context` of `[]`?